pyc_code.h

```diff
@@ -87,6 +87,10 @@ class PycCode : public PycObject {
         m_globalsUsed.emplace_back(std::move(varname));
     }
 
+    typedef std::tuple<int, int, int, int, bool> exception_table_entry_t;
```

Comment:
I would prefer to see an actual struct here; otherwise the elements have no obvious meaning...